app/src/main/java/com/android/unio/model/event/EventCard.kt

```diff
@@ -0,0 +1,139 @@
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.android.unio.R
+
+@Composable
+fun EventCard(event: Event, onClick: () -> Unit) {
+  val backgroundColor = Color(0xFF2596BE)
+  val backgroundImage = painterResource(id = R.drawable.photo_2024_10_08_14_57_48)
+
+  Column(
+      modifier =
+          Modifier.fillMaxWidth()
+              .padding(vertical = 8.dp)
+              .clickable(onClick = onClick)
+              .testTag("event_EventListItem")
+              .clip(RoundedCornerShape(10.dp))
+              .background(Color(0xFFF0ECF4))) {
+        Image(
+            painter = DynamicImage(event.image),
+            contentDescription = null,
+            modifier =
+                Modifier.fillMaxWidth()
+                    .height(100.dp)
+                    .clip(RoundedCornerShape(topStart = 10.dp, topEnd = 10.dp))
+                    .testTag("event_EventImage"),
+            contentScale = ContentScale.Crop // crop the image to fit
+            )
+
+        Column(modifier = Modifier.padding(horizontal = 10.dp, vertical = 5.dp)) {
+          Row(verticalAlignment = Alignment.CenterVertically, modifier = Modifier.fillMaxWidth()) {
+            Row(verticalAlignment = Alignment.CenterVertically, modifier = Modifier.weight(1f)) {
+              Text(
+                  modifier =
+                      Modifier.padding(vertical = 1.dp, horizontal = 4.dp)
+                          .testTag("event_EventTitle")
+                          .wrapContentWidth(), // Make sure the text only takes as much space as
+                  // needed
+                  text = event.title,
+                  style = MaterialTheme.typography.titleMedium,
+                  fontWeight = FontWeight.Bold,
+                  color = Color.Black)
+
+              Spacer(modifier = Modifier.width(6.dp))
+
+              Box(
+                  modifier =
+                      Modifier.clip(RoundedCornerShape(4.dp))
+                          .background(addAlphaToColor(event.types.get(0).color, 200))
+                          .wrapContentWidth()) {
+                    Text(
+                        text = event.types.get(0).text,
+                        modifier =
+                            Modifier.padding(horizontal = 4.dp, vertical = 4.dp)
+                                .testTag("event_EventMainType"),
+                        color = Color.Black,
+                        style = TextStyle(fontSize = 8.sp))
+                  }
+            }
+
+            Spacer(modifier = Modifier.width(6.dp))
+
+            Image(
+                painter = painterResource(id = R.drawable.clic),
+                contentDescription = null,
+                modifier =
+                    Modifier.size(24.dp)
+                        .align(Alignment.CenterVertically)
+                        .testTag("event_ClicImage"))
+          }
```

Comment:
Even though we can't show multiple association icons yet, the Icon should be the icon of the event's first association instead of a placeholder one. It would of course only work with the example event you use because the image loading is to be implemented.

Reply:
Ok ! I will link it to the icon then !

Reply:
Wait I don't agree with the fact that it should only be the first association's logo